src/indexer_worker.ts

```diff
@@ -228,26 +238,35 @@ export class Indexer {
     //if (this.config.debug) console.debug(query.toParams()); // DEBUG
     try {
       await this.pgClient.query(query.toParams());
-      await this.pgClient.query(`NOTIFY log, '${JSON.stringify({blockId: blockID, index: transactionIndex})}'`);
+      await this.pgClient.query(
+        `NOTIFY log, '${JSON.stringify({
+          blockId: blockID,
+          index: transactionIndex,
+        })}'`
+      );
     } catch (error) {
       console.error('indexEvent', error);
     }
   }
 
   async notifyNewHeads(blockID: number): Promise<void> {
-    if (this.pendingHeadBlock == 0) { this.pendingHeadBlock = blockID }
+    if (this.pendingHeadBlock == 0) {
+      this.pendingHeadBlock = blockID;
+    }
     for (;;) {
-      if ( await this.isBlockIndexed(this.pendingHeadBlock)) {
+      if (await this.isBlockIndexed(this.pendingHeadBlock)) {
         this.pgClient.query(`NOTIFY block, '${this.pendingHeadBlock}'`);
-        this.pendingHeadBlock +=  1;
+        this.pendingHeadBlock += 1;
       } else {
-        return
+        return;
       }
     }
   }
   async isBlockIndexed(blockID: number): Promise<boolean> {
-    const result = await this.pgClient.query(`SELECT 1 FROM block WHERE id = ${ this.pendingHeadBlock } LIMIT(1)`)
-    return result.rows.length == 1
+    const result = await this.pgClient.query(
+      `SELECT 1 FROM block WHERE id = ${this.pendingHeadBlock} LIMIT(1)`
```

Comment:
This could be passing a parameter instead of doing interpolation.

Reply:
Deffinatly. This was already addressed in [Clean up the WebSockets implementation](https://github.com/aurora-is-near/aurora-relayer/pull/142). Will be resolved after rebase.